Pages/ProjectPages/ProjectListPage.cs

```diff
@@ -0,0 +1,61 @@
+﻿using OpenQA.Selenium;
+using System.Collections.ObjectModel;
+using TMS_Selenium.Element;
+
+namespace TMS_Selenium.Pages.ProjectPages
+{
+    public class ProjectListPage :BasePage
+    {
+        private static readonly By _messageBy = By.CssSelector("[data-testid='messageSuccessDivBox']");
+        private static readonly By _addProgectButton = By.XPath("//a[contains(text(), 'Add Project')]");
+        private static readonly By _projectsQuantityBy = By.ClassName("\"text-softer\"");
+        private string _endPoint = "index.php?/admin/projects/overview";
+        public ProjectListPage(IWebDriver driver, bool openPageByUrl = false) : base(driver, openPageByUrl)
+        {
+        }
+        public string GetMessage() => new Message(driver, _messageBy).Text.Trim();
+        public Button AddProjectButton() => new Button(driver, _addProgectButton);
+        public string GetProjectQuantity() => new Message(driver, _projectsQuantityBy).Text.Trim();

```

Comment:
it is possible to use here int 
public int GetProjectQuantity() {
        int.TryParse(new Message(driver, _messageBy).Text.Trim(), out int result);
        return result;
        }
         int.TryParse will try to convert your string to int and return result into 'result' variable